### PR TITLE
nautilus: mgr/dashboard: code documentation

### PR DESCRIPTION
**Pull request by LenzGr**
http://tracker.ceph.com/issues/39345

### CODE REVIEW

File: src/pybind/mgr/dashboard/HACKING.rst [hunk 1]
```diff
@@ -65,6 +65,15 @@ Run ``npm run build`` to build the project. The build artifacts will be
 stored in the ``dist/`` directory. Use the ``-prod`` flag for a
 production build. Navigate to ``https://localhost:8443``.
 
+Build the Code Documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run ``npm run doc-build`` to generate code docs in the ``documentation/``
+directory. To make them accesible locally for a web browser, run
+``npm run doc-serve`` and they will become available at ``http://localhost:8444``.
+With ``npm run compodoc -- <opts>`` you may
```
Comment (p-na) [inline src/pybind/mgr/dashboard/HACKING.rst]: Of course too late to mention that in that backport pr, but `npx compoc` can interchangeably be used for `npm run compodoc --` as well as for all other packages installed by npm, like `npx jest`.

**Comment by s0nea** [comment]
jenkins test docs

**Comment by smithfarm** [comment]
@LenzGr Can you reproduce the error shown in https://tracker.ceph.com/issues/39556 ?

AFAICT the steps to reproduce are:

1. clone ceph/ceph.git, checkout master
2. `./make-dist`

**Comment by yuriw** [comment]
[wip-yuri2-testing-2019-05-16-1555-nautilus](https://trello.com/c/VvCEDimT/616-wip-yuri2-testing-2019-05-16-1555-nautilus)

**Comment by smithfarm** [comment]
jenkins re-test this please

**Pull request merged**